RELEASE-NOTES.md

```diff
@@ -0,0 +1,51 @@
+# node_package 3.0.0
+## Security Improvements
+### Introduction
+### Security Advisory dated March 1, 2016
+It was [recently reported](http://docs.basho.com/riak/latest/community/product-advisories/codeinjectioninitfiles/) that, if a user could gain access to the `riak` user (or, in node_package parlance, the `package_install_user`), that use would then have write access to init scripts that are generally run as `root`, exposing an escalation of privileges attack where said use could then get the `root` user to execute a script that could allow the original user to become `root` on the system.
+
+### Additional Security Review
+
+After the security advisory was initially released, a more through review of all of the `node_package`-generated packages was conducted. This review found some additional cases of files or directories owned by the `package_install_user` or `package_install_group` that could also potentially allow a user with access to run in the context of that user to overwrite files that may later be executed by `root`. This release of `node_package` has significantly tightened the ownership and permissions of files installed, in most cases following the target systems' conventions (`root:root`, `root:bin`, `root:wheel`) for all files that are executable or could be executed, including library files that the packaged application may read.
```

Comment:
through -> thorough "a more thorough review" 
" a user with access" ... do we mean a user with write-access? I would change that section to: "... allow a user with access to `riak` user to overwrite files..." (if that makes sense).